python/xorbits/_mars/dataframe/window/rolling/aggregation.py

```diff
@@ -243,7 +243,8 @@ def _find_extra_chunks_for_int_window(cls, op, inp, cur_chunk_index):
 
         # find prev chunks
         i = cur_chunk_index[axis]
-        rest = window if not center else window // 2
+        # handle all the possibilities of op.closed.
+        rest = window if not center else window // 2 + 1
```

Comment:
Have you added a test to cover this corner case?

Reply:
Yes, these lines are covered when op.closed being set to values other than 'right'.